### ISSUE TITLE
Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

### ISSUE THREAD

**Issue by Cesia02** [codesandbox/codesandbox-client#4374]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 78.0.3904 | Windows 8.1           | 5e124550f          |

  **Route:**
  https://codesandbox.io/s/quirky-meadow-g6ybr?file=/index.html

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
  Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at ys (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1753033)
    at ms (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1751034)
    at ys (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1752770)
    at yl (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1772035)
    at https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1793567
    at qo (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1717940)
    at bl (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1769920)
    at il (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1760247)
    at https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1718231
    at https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1793567
    at qo (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1717940)
    at Yo (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1718176)
    at $o (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1718111)
    at ce (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1787522)
    at On (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1702069)
    at Mn (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1703026)
    at Tn (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1702314)
    at https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1793567
    at jn (https://codesandbox.io/static/js/common.dada56d39.chunk.js?_sw-precache=5609f1a4c0a7be0a60682b7d9cad63e8:1:1701790)
  
    in elements__Container
    in div
    in dc
    in div
    in Styled(styled.div)
    in li
    in Entry___StyledListAction
    in div
    in n
    in div
    in Unknown
    in DragSource(Component)
    in div
    in elements__EntryContainer
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Kc
    in div
    in elements__Opener
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Styled(styled.div)
    in section
    in Styled(styled.div)
    in Unknown
    in Files___StyledCollapsible
    in hl
    in is
    in div
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in c
    in t
  ```

  </details>

**Comment by garethx** [comment]
@Cesia02 What were you trying to accomplish when the crash occurred?

**Comment by SaraVieira** [comment]
closed by #4571
